### DISCUSSION TITLE
Add warning/label below language selection for community translation

### DISCUSSION THREAD

**Discussion by lilyremigia** [ppy/osu#21933] [Ideas] [answered]
Add a warning/label below language selection, that they're community translations and may be incorrect, and when clicked leads to the Crowdin page.

**Comment by bdach** [comment]
Why is this a problem exactly? The website doesn't do any of this and it also sources translation from crowdin.

**Comment by frenzibyte** [comment]
Adding a link to Crowdin that I'm fine with (increases visibility for those looking to help), but explicitly mentioning that they "may be incorrect" sounds like bad wording. I would put something along the lines of "You can help in improving these translations [here](https://crowdin.com/project/osu-web)" or even "These translations are openly editable and are sourced from [here](https://crowdin.com/project/osu-web)".

**Comment by peppy** [comment, marked as answer]
Not happening. Instead please fix the translations.
